src/nanovg_gl.h

```diff
@@ -147,6 +147,7 @@ struct GLNVGtexture {
 	int width, height;
 	int type;
 	int flags;
+	unsigned int salt;
```

Comment:
Isn't salt already stored as part of the id?

Reply:
Well... thinking about it.  The goal is to make the ID == the index in the array, so storing the ID in the Texture structure is no longer necessary.

The salt is what need to be stored.  The ID returned to the application is the ID + the salt.